imageprocessor/views.py

```diff
@@ -20,15 +25,29 @@ def tagged_pictures(request):
 
 def classify(request):
 	context = {}
-	form = ImageForm(request.POST or None)
+	form = ImageForm(request.POST or None, request.FILES or None)
 	if request.method == 'POST':
 		try:
-			image_file = request.FILES.get('file')
+			image_file = request.FILES['file']
```

Comment:
This line fails when I try to upload an image using postman. It was doing this before, but I was wondering if you have any idea why this might be happening?

Reply:
I am guessing it has to do with the csrf token, however when i tried to remove it that caused a different error

Reply:
I fixed that by adding the @csrf_exempt annotation, but then when it tried to get the file off the request it throws a MultiValueDictKeyError